sdk/tables/Azure.Data.Tables/tests/TableConnectionStringTests.cs

```diff
@@ -213,36 +219,38 @@ public void ParseAzuriteConnString()
-            yield return new object[] { new Uri($"https://{AccountName}.table.cosmos.azure.com:443/{TableName}/") };
-            yield return new object[] { new Uri($"https://{AccountName}.table.cosmos.azure.com:443/Tables('{TableName}')/") };
-            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/{TableName}") };
-            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/{TableName}/") };
-            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/Tables('{TableName}')/") };
-            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{TableName}") };
-            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{TableName}/") };
-            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/Tables('{TableName}')/") };
-            yield return new object[] { new Uri($"https://10.0.0.1:10002/{AccountName}/{TableName}") };
-            yield return new object[] { new Uri($"https://10.0.0.1:10002/{AccountName}/{TableName}/") };
-            yield return new object[] { new Uri($"https://10.0.0.1:10002/{AccountName}/Tables('{TableName}')/") };
+            yield return new object[] { new Uri($"https://{AccountName}.table.cosmos.azure.com:443/{TableName}"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://{AccountName}.table.cosmos.azure.com:443/{TableName}/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://{AccountName}.table.cosmos.azure.com:443/Tables('{TableName}')/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/{TableName}"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/{TableName}/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://{AccountName}.table.core.windows.net/Tables('{TableName}')/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{AccountName}"), AccountName, AccountName };
+            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{AccountName}/"), AccountName, AccountName };
+            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{TableName}"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/{TableName}/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://127.0.0.1:10002/{AccountName}/Tables('{TableName}')/"), AccountName, TableName };
+            yield return new object[] { new Uri($"https://10.0.0.1:10002/{AccountName}/{TableName}"), AccountName, TableName };
```

Comment:
Should we consider using `localhost` for any of these just to ensure that there's no dependency on the host having embedded `.` characters?

Reply:
added